packages/styled-components/src/models/StyledNativeComponent.js

```diff
@@ -219,20 +206,22 @@ export default (InlineStyle: Function) => {
       },
     });
 
-    if (isClass) {
-      hoist(WrappedStyledNativeComponent, (target: any), {
-        // all SC-specific things should not be hoisted
-        attrs: true,
-        displayName: true,
-        shouldForwardProp: true,
-        inlineStyle: true,
-        styledComponentId: true,
-        target: true,
-        withComponent: true,
-      });
-    }
+    // $FlowFixMe
+    WrappedStyledComponent.toString = () => `.${WrappedStyledComponent.styledComponentId}`;
```

Comment:
I believe this is obsolete however

Reply:
Yes, native doesn't have component selectors